### PR TITLE
Add the HTML structure and styles for the calculator UI

### PR DESCRIPTION
**Pull request by Ellon-M**
## Changes in this PR

- Created a `Calculator.js` file to render the JSX for the calculator user interface.
- Added the calculator component created to the `App.js` file.
- Wrote styles for the calculator in `index.css`.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
-      <h3>Math Magicians!</h3>
+      <Calculator />
     </div>
   );
```
Comment (youmari) [inline src/App.js]: - [ ] it would be better if `App.js` also class-based in order to complete the requirements

File: src/index.js [hunk 2]
```diff
@@ -2,16 +2,10 @@ import React from 'react';
...
 root.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
 );
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
```
Comment (youmari) [inline src/index.js:17]: - [ ] As required for the project, kindly make sure you delete all the boilerplate from CRA (text, images, styles,...). You should also delete the reportWebVitals.js, setupTests.js,... Make sure you delete all the non related files of your project.

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Ellon-M , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @Ellon-M , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**